x/leverage/keeper/collateral.go

```diff
@@ -52,3 +52,15 @@ func (k Keeper) setCollateralAmount(ctx sdk.Context, borrowerAddr sdk.AccAddress
 	}
 	return nil
 }
+
+// GetTotalCollateral returns an sdk.Coin representing how much of a given uToken
+// the x/leverage module account currently holds as collateral.
+func (k Keeper) GetTotalCollateral(ctx sdk.Context, denom string) sdk.Int {
+	if !k.IsAcceptedUToken(ctx, denom) {
+		// non-uTokens cannot be collateral
+		return sdk.ZeroInt()
```

Comment:
shouldn't we return error?

Reply:
That is an option, yes.

The current approach follows the other getters, in that they return zero when queried for the borrowed/reserved/etc amount of something that doesn't exist or wouldn't be valid input for a setter.

Compare to querying the borrowed amount of an invalid denom (or unregistered Token, or a uToken denom) - it will simply return zero. Actually trying to borrow the same denom returns an error though.

Reply:
OK, then let's update the function docs please.

Reply:
lmk if the current godoc looks good

Reply:
Looks good.